pygmt/tests/test_datasets_earth_relief.py

```diff
@@ -83,6 +83,24 @@ def test_earth_relief_05m_without_region():
         load_earth_relief("05m")
 
 
+def test_earth_relief_03s_landonly_srtm():
+    """
+    Test loading original 3 arc-second land-only SRTM tiles.
+    """
+    data = load_earth_relief(
+        "03s", region=[135, 136, 35, 36], registration="gridline", use_srtm=True
+    )
```

Comment:
Do we want to cache this slice of a 3 arc second grid in our CI? I.e. change these lines https://github.com/GenericMappingTools/pygmt/blob/65b099a5873b4f9e76d7fccdbc8882d9b37870e3/.github/workflows/cache_data.yaml#L33-L36

I see at https://docs.generic-mapping-tools.org/6.1/datasets/remote-data.html#id1 that @earth_relief_03s is 6.8GB so maybe we should cache only the tiles that are needed.

Reply:
The tile downloaded by this example is about 900 k.
```
$ du -h ~/.gmt/server/earth/earth_relief/earth_relief_03s_g 
900K    ~/.gmt/server/earth/earth_relief/earth_relief_03s_g
```

Reply:
Yes, we need to cache this tile.

Reply:
Do I have to revise the workflow in this PR? @seisman @weiji14

Reply:
Please open a separate PR for it. See https://github.com/GenericMappingTools/pygmt/pull/844 for an example. Please note that the `cache_data.yaml` workflow only runs weekly, so you need to uncomment the following line to trigger the workflow in PR so that the caches are updated, and then comment it again. 

https://github.com/GenericMappingTools/pygmt/blob/905ff620209f106e53112ef068afcd70e1c36354/.github/workflows/cache_data.yaml#L6

Reply:
@weiji14 @seisman done in #985.